src/components/organisms/page/ports/vim/SectionPluginSupport/SectionPluginSupport.jsx

```diff
@@ -106,8 +106,7 @@ const SectionPluginSupport = ({ assets }) => (
         </Visualization>
         <Text>
           <Subline>
-            Beautiful syntax highlighting with support for gr
-            <Link href="https://github.com/fatih/vim-go">vim-go</Link> and{" "}
+            Beautiful syntax highlighting with support for <Link href="https://github.com/fatih/vim-go">vim-go</Link>{" "}
```

Comment:
Ah just checked the deployment preview and saw it: The "and" between the both links is missing.